Editing_Guidelines.md

```diff
@@ -60,6 +60,12 @@ Dr. Arach is a complete weirdo, and something of a *mad biologist*. Conveying he
 
 **Sample sentence:** "Oooh, sublime sugar! You're pervading all five of my senses! You're even making my uterus and my ovaries melt.. I feel it coming! Haah.. Thanks for the meaaal!" (*filename 02_00_ARC02_1B, page 39, line 2802*)
 
+Arach has many different designators. For consistency, use:
+阿良句先生 - Arach-sensei
+阿良句医師 - Dr. Arach
+阿良句博士 - Prof. Arach
```

Comment:
Note that we're using "professor" here because of the following lines in day 6:
```
sha:1d1d95ce0e45595f743d1dc8f38dc986803e85e7]{
-- Page 238, Offset 14489. Mods: @k, @e.
-- 「いいのいいの。アタシ、しょせん無免かつ専門外だし？　本職はカガク者だし？
"Don't mention it. Besides, I'm not even licensed and this is outside my area of expertise. I'm really more of a scientist.
}
[sha:4cc43f5eb89dc608704245bb8f7d1cf2dfce9903]{
-- Page 238, Offset 14490. Glued. Mods: @x.
-- 志貴チャン、博士って呼んでくれたし？」
// Leading space for glue
 Shiki-chan called me a professor after all."
}
```
It was originally suggested that we keep this as "doctor" but カガク者 makes that a little confusing